client/gui/CAnimation.h

```diff
@@ -161,17 +161,21 @@ class CFadeAnimation
 public:
 	enum class EMode
 	{
-		NONE, IN, OUT
+		NONE,
+		IN,
+		OUT

```

Comment:
I'd add trailing comma after the last element

Reply:
I'd also add them but apparently this thing only works for JavaScript...
At least it seems that it will preserve any pre-existing trailing commas and won't attempt to remove them

Reply:
But JSON parser of Java does not like it)

Reply:
how is C++ header related to Java or JSON?